configserver/src/main/java/com/yahoo/vespa/config/server/deploy/ModelContextImpl.java

```diff
@@ -183,6 +184,8 @@ public Properties(ApplicationId applicationId,
             this.endpointCertificateSecrets = endpointCertificateSecrets;
             defaultTermwiseLimit = Flags.DEFAULT_TERM_WISE_LIMIT.bindTo(flagSource)
                     .with(FetchVector.Dimension.APPLICATION_ID, applicationId.serializedForm()).value();
+            defaultSoftStartSeconds = Flags.DEFAULT_SOFT_START_SECONDS.bindTo(flagSource)
+                    .with(FetchVector.Dimension.APPLICATION_ID, applicationId.serializedForm()).value();
```

Comment:
Since only the application ID dimension is set here, the value can only depend on the application ID, not hostname.

Reply:
So how can I control this with hostname too ?
I see that there is a Dimension.HOSTNAME, but I do not have the hostname accessible here.
Any example ?

Reply:
All current examples of HOSTNAME are trivial: the hostname would be given and you would invoke `with(HOSTNAME, hostname)` here like you did for APPLICATION_ID.

However this will not work here, as all we have are data structures that will apply during the deployment of a given application, irrespective of the hosts involved.

So could we wait with resolving the flag until closer to the place in the config model where we know the involved host?  No, because changing the flag wouldn't bump the config version and wouldn't trigger a new fetch, and possibly leaving half of the nodes with one view of the flags and the other half with another view.

One possibility is to freeze the flag values (fork the flag source), and pass that through to the config model. Another faster time-to-market would be to have a special value (say a String) which is specially handled in the config model (split on ";" in to a set of hostnames).

Reply:
OK, then the host dimension needs another solution.